starknet_devnet/blueprints/rpc/structures/payloads.py

```diff
@@ -622,6 +623,15 @@ def function_abi_entry(abi_entry: AbiEntryType) -> FunctionAbiEntry:
     """
     Convert function gateway abi entry to rpc FunctionAbiEntry
     """
+    if "stateMutability" in abi_entry:
+        return FunctionAbiEntry(
```

Comment:
Link from Slack: https://stackoverflow.com/questions/72326834/how-to-have-optional-keys-in-typeddict

Reply:
Addressed here [352b608](https://github.com/0xSpaceShard/starknet-devnet/pull/494/commits/352b6084319b6bea9fb081fa846444cddd6f07a3)

Reply:
Actually my idea was that maybe we don't have to have the object initialization in two places, rather have a single initialization and conditional adding of state mutability if in abi_entry. But if it can't be done, then let's keep it like this.

Reply:
Updated here [51a18ee](https://github.com/0xSpaceShard/starknet-devnet/pull/494/commits/51a18ee8eb81577d818448ba8ac516dc66369c81)